### PR TITLE
Integrate lazy initialization to be integrated with viewer

### PR DESCRIPTION
**Pull request by xepozz**
https://github.com/yiisoft/yii-debug-viewer/pull/61

### CODE REVIEW

File: packages/yii-dev-panel/src/bootstrap.tsx [hunk 1]
```diff
@@ -5,9 +5,25 @@ import '@yiisoft/yii-dev-panel/index.css';
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 
-const root = ReactDOM.createRoot(document.getElementById('root') as HTMLElement);
-root.render(
-    <React.StrictMode>
-        <App />
-    </React.StrictMode>,
+(function YiiDevPanelWidget(scope) {
+    console.log('set widget', scope);
```
Comment (samdark) [inline packages/yii-dev-panel/src/bootstrap.tsx:9]: Should be removed before merge?

File: packages/yii-dev-panel/src/bootstrap.tsx [hunk 2]
```diff
@@ -5,9 +5,25 @@ import '@yiisoft/yii-dev-panel/index.css';
...
+    scope.init = function () {
+        console.log('this', this);
```
Comment (samdark) [inline packages/yii-dev-panel/src/bootstrap.tsx:11]: Should be removed before merge?

File: packages/yii-dev-panel/vite.config.ts [hunk 3]
```diff
@@ -46,11 +46,11 @@ export default defineConfig(async ({command}) => {
             }),
             viteTsconfigPaths(),
             svgrPlugin(),
-            federation({
-                name: 'host',
-                remotes: {},
-                shared: sharedModules,
-            }),
+            // federation({
```
Comment (samdark) [inline packages/yii-dev-panel/vite.config.ts:49]: Should be removed before merge?

Comment (xepozz) [reply packages/yii-dev-panel/vite.config.ts]: nope

**Pull request merged**